### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hsubramony**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ pip install git+https://github.com/huggingface/optimum-habana.git
 > ```bash
 > git clone https://github.com/huggingface/optimum-habana.git
 > cd optimum-habana
-> python setup.py install
+> pip install -e .
 > ```
```
Comment (regisss) [inline README.md:57]: I would actually remove the whole comment from line 52 to line 57. `python setup.py install` doesn't work while `pip install` does.

**Review by regisss — COMMENTED** [review]
Hi @hsubramony! Thanks for this PR. As stated in the comment, I think we should just remove this part of the README.

File: README.md (hunk 1 above)
Comment (hsubramony) [reply README.md:57]: got it ..will do

**Review by regisss — APPROVED** [review]
LGTM, thanks!

**Pull request merged**